integration-tests/src/test/java/com/datastax/oss/driver/core/cql/SecondTraceIT.java

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Let's find better place for this suite, i am thinking about datastax/oss/driver/core/dns/DNSEndpointsIT.java`